cli/src/pcluster/templates/cluster_stack.py

```diff
@@ -747,33 +738,36 @@ def _add_efs_mount_target(
                 )
             checked_availability_zones.append(availability_zone)
 
-    def _add_raid_volume(self, id: str, shared_ebs: SharedEbs):
-        """Add specific Cfn Resources to map the EBS storage."""
-        # TODO: add RAID storage
-        return f"fake_id{id}-{shared_ebs.mount_dir}"
+    def _add_raid_volume(self, id_prefix: str, shared_ebs: SharedEbs):
+        """Add specific Cfn Resources to map the RAID EBS storage."""
+        ebs_ids = []
+        for index in range(shared_ebs.raid.number_of_volumes):
+            ebs_id = self._add_cfn_volume(id_prefix + f"Volume{index}", shared_ebs)
```

Comment:
`id_prefix + f"Volume{index}"` --> `f"{id_prefix}Volume{index}"`

then you can avoid defining the ebs_id variable:
`ebs_ids.append(self._add_cfn_volume(f"{id_prefix}Volume{index}", shared_ebs))`

Reply:
Done